packages/react-components/theme-designer/src/components/ColorTokens/ColorTokens.tsx

```diff
@@ -0,0 +1,138 @@
+      colorCompoundBrandForeground1Pressed: 80,
+      colorBrandForeground1: 80,
+      colorBrandForeground2: 70,
+      colorBrandForegroundInverted: 100,
+      colorBrandForegroundInvertedHover: 110,
+      colorBrandForegroundInvertedPressed: 100,
+      colorBrandForegroundOnLight: 80,
+      colorBrandForegroundOnLightHover: 70,
+      colorBrandForegroundOnLightPressed: 50,
+      colorBrandForegroundOnLightSelected: 60,
+      colorBrandBackground: 80,
+      colorBrandBackgroundHover: 70,
+      colorBrandBackgroundPressed: 40,
+      colorBrandBackgroundSelected: 60,
+      colorCompoundBrandBackground: 80,
+      colorCompoundBrandBackgroundHover: 70,
+      colorCompoundBrandBackgroundPressed: 60,
+      colorBrandBackgroundStatic: 80,
+      colorBrandBackground2: 160,
+      colorBrandBackgroundInvertedHover: 160,
+      colorBrandBackgroundInvertedPressed: 140,
+      colorBrandBackgroundInvertedSelected: 150,
+      colorNeutralStrokeAccessibleSelected: 80,
+      colorCompoundBrandStroke: 80,
+      colorCompoundBrandStrokeHover: 70,
+      colorCompoundBrandStrokePressed: 60,
+    };
+  } else {
+    // theme = createLightTheme(brand);
+    initialColors = {
+      colorNeutralForeground2BrandHover: 80,
+      colorNeutralForeground2BrandPressed: 70,
+      colorNeutralForeground2BrandSelected: 80,
+      colorNeutralForeground3BrandHover: 80,
+      colorNeutralForeground3BrandPressed: 70,
+      colorNeutralForeground3BrandSelected: 80,
+      colorBrandForegroundLink: 70,
+      colorBrandForegroundLinkHover: 60,
+      colorBrandForegroundLinkPressed: 40,
+      colorBrandForegroundLinkSelected: 70,
+      colorCompoundBrandForeground1: 80,
+      colorCompoundBrandForeground1Hover: 80,
+      colorCompoundBrandForeground1Pressed: 80,
+      colorBrandForeground1: 80,
+      colorBrandForeground2: 70,
+      colorBrandForegroundInverted: 100,
+      colorBrandForegroundInvertedHover: 110,
+      colorBrandForegroundInvertedPressed: 100,
+      colorBrandForegroundOnLight: 80,
+      colorBrandForegroundOnLightHover: 70,
+      colorBrandForegroundOnLightPressed: 50,
+      colorBrandForegroundOnLightSelected: 60,
+      colorBrandBackground: 80,
+      colorBrandBackgroundHover: 70,
+      colorBrandBackgroundPressed: 40,
+      colorBrandBackgroundSelected: 60,
+      colorCompoundBrandBackground: 80,
+      colorCompoundBrandBackgroundHover: 70,
+      colorCompoundBrandBackgroundPressed: 60,
+      colorBrandBackgroundStatic: 80,
+      colorBrandBackground2: 160,
+      colorBrandBackgroundInvertedHover: 160,
+      colorBrandBackgroundInvertedPressed: 140,
+      colorBrandBackgroundInvertedSelected: 150,
+      colorNeutralStrokeAccessibleSelected: 80,
+      colorCompoundBrandStroke: 80,
+      colorCompoundBrandStrokeHover: 70,
+      colorCompoundBrandStrokePressed: 60,
+    };
+  }
+
+  const colorReducer: (state: AccentColors, action: { colorToken: string; newValue: Brands }) => AccentColors = (
```

Comment:
i think this should only include a list of overrides. otherwise you won't know what is returned from the default theme, and what is being overridden.